__tests__/components/button/__snapshots__/Dropdown.test.tsx.snap

```diff
@@ -30,58 +30,40 @@ exports[`Dropdown should handle sizes correctly 1`] = `
     >
       <ul>
         <li>
-          <a
-            href="https://fd.nl"
-            rel="noopener noreferrer"
-            target="_blank"
-          >
+          <span>
```

Comment:
Why is the link changed to a span?
How does this affect the function of the dropdown and what happens in browsers for the seeing-impaired (NVDA)?